src/components/AddItem/AddItem.component.jsx

```diff
@@ -57,14 +64,16 @@ const AddItem = () => {
 
 
 
-        console.log('items from add item:', items);
-
         if (!items.includes(cleanInput)) {
           return firebase.dataBase.collection(collectionTokenName).add({
             name: itemName,
             resupplyPeriod: resupplyPeriod,
             id: itemId,
             lastPurchaseDate: lastPurchaseDate,
+            lastEstimate: lastEstimate,
+            latestInterval: latestInterval,
+            numberOfPurchases: numberOfPurchases,
+            nextPurchaseInterval: nextPurchaseInterval,
```

Comment:
One thing that you can do to write less code in the future is make the state variable match the label string, so you can write out your object with shorthand like so:
```suggestion
            name,
            resupplyPeriod,
            id,
            lastPurchaseDate,
            lastEstimate,
            latestInterval,
            numberOfPurchases,
            nextPurchaseInterval,
```
Doesn't need to be changed. Just a helpful tip for your future projects.